nesta/packages/dq/general.py

```diff
@@ -0,0 +1,132 @@
+import pandas as pd
+import numpy as np
+import itertools
+
+def missing_values(input):

```

Comment:
There could be many more types of null values than those identified by the dataframe. It could be good to have an arg that lets the user specify a list of values that they know to represent missing data.